File: Models/Song.cs
namespace AboutMe.Models
{
	public class Song
    {
        public int Id { get; set; }
        public string SongName { get; set; }
        public string Artist { get; set; }
        public string Album { get; set; }
        public string Genre { get; set; }
        public string Time { get; set; }
        public string AppleMusicLink { get; set; }
        public string SpotifyLink { get; set; }
        public int IdYear { get; set; }
    }
}

File: api/app/schemas/song.py
from models.about_me_model import AboutMeModel
import datetime


class SongSchema(AboutMeModel):
    song_name: str = None
    artist: str = None
    album: str = None
    genre: str = None
    song_length: datetime.time = None
    apple_music_link: str = None
    spotify_link: str = None
    year: int = None
